README.md

```diff
@@ -105,50 +107,53 @@ Use [cron-parser](https://github.com/harrisiirak/cron-parser).
 └───────────────────────── second (0 - 59, optional)
 ```
 
-example:
+Example:
+
 ```javascript
-// exce task every 3 hours
+// To execute task every 3 hours
 exports.schedule = {
   type: 'worker',
-  cron: '0 0 */3 * * *',
+  cron: '0 0 */3 * * *'
```

Comment:
using trailing comma, so:
- when add item to array,  diff will only one line
- and git blame of line will not got wrong